tests/nn/misc/test_checkpoint_activations.py

```diff
@@ -16,18 +14,27 @@
 from fairscale.nn.misc.checkpoint_activations import checkpoint_wrapper
 
 
+def get_cuda_mem_allocated():
+    """Helper to get cuda memory allocated if possible."""
+    if torch.cuda.is_available():
+        return torch.cuda.memory_allocated()
+    else:
+        return 0
+
+
 class Model(nn.Module):
-    def __init__(self, use_pytorch_checkpoint=False, use_fairseq_checkpoint=False, **kwargs):
+    def __init__(self, use_pytorch_checkpoint=False, use_fairscale_checkpoint=False, **kwargs):
         super().__init__()
-        torch.manual_seed(0)
+        torch.manual_seed(0)  # make sure weights are deterministic.
+        assert not (use_pytorch_checkpoint and use_fairscale_checkpoint), "Doesn't make sense to use both"
```

Comment:
Maybe a more descriptive error message "Cannot use both pytorch and fairscale checkpointing mechanisms."?

Reply:
done